CONTRIBUTING.md

```diff
@@ -41,6 +46,16 @@ As with other types of contributions, the first step is to [**open an issue on G
 
 Once you've opened an issue, check out our [Developer Guide](./DEVELOPER_GUIDE.md) for instructions on how to get started.
 
+_Note_: Since you are a beginner, we advise you to start with simple issues.
+
+For this there is a special label "<font color="SlateBlue">**Good first issue**</font>"
+
+Warning! If you have selected an issue and want to get started, check if the issue is free. If for it has already been assigned <br> and there is no pull request for a long time, ask that user using @ (e.g @nickname) if he/she is willing to concede to you.
+
+If the issue has recently been created and there are no appointees, then feel free to write in a comment that you are ready to take on the issue.
+
+Feel free to create an issue if you have not been able to work out the installation, or a bug within the project. Also feel free to ask on the issue page if you are solving a problem but can't figure out the structure of the project.
+
```

Comment:
This whole topic should be moved to the "Contributing Code" section. The idea is that opening and participating in issues is one type of contribution, separate from providing code.